charts/stardog/templates/configmap.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "stardog.fullname" . }}
+  name: {{ include "stardog.fullname" . }}-log4j
```

Comment:
split the configmap down to just a file, in a specific place, to avoid the `cp` in the init .sh